data/snippets.conf

```diff
@@ -19,6 +19,13 @@
 
 # Default is used for all filetypes and keys can be overwritten by [filetype] sections
 [Default]
+if=if (%cursor%)%block_cursor%
+else=else%block_cursor%
+for=for (i = 0; i < %cursor%; i++)%block_cursor%
+while=while (%cursor%)%block_cursor%
+do=do\n{\n\t%cursor%\n} while (%cursor%)\n%cursor%
+switch=switch (%cursor%)%brace_open%case %cursor%:\n\t\t%cursor%\n\t\tbreak;\n\tdefault:\n\t\t%cursor%\n%brace_close%%cursor%
+try=try%block%\ncatch (%cursor%)%block_cursor%
```

Comment:
I don't have think about it, it was on my previous installation of Geany and this is inside Ubuntu configuration, not now. But sure it's finally stupid to put it in default section.